scripts/shared/post_mortem.sh

```diff
@@ -59,6 +59,9 @@ function post_analyze() {
     print_section "* Output of 'subctl diagnose all' in $cluster *"
     subctl diagnose all --context "$cluster"
 
+    print_section "* Collecting 'subctl gather' in $cluster *"
+    subctl gather --context "$cluster"
```

Comment:
Let's specify the output directory to make it deterministic.

```suggestion
    subctl gather --context "$cluster" --dir gather_output
```

Reply:
Cool, good idea.